gdrive

```diff
@@ -63,6 +65,12 @@ def get_filename(service, file_id):
     print('Filename is "%s" for ID: %s.' % (filename, file_id))
     return filename
 
+def sizeof_fmt(num, suffix='B'):
```

Comment:
what's fmt? please rename to size_of_**full_meaning**

Reply:
renamed to size_to_human_readable